PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -305,29 +310,24 @@ private static string ToCommandName(Guid commandGroup, uint commandId)
         /// </summary>
         private void TriggerCompletion()
         {
-            
-            
             _completionCaretPosition = (int)_textView.Caret.Position.BufferPosition;
-            Tasks.Task.Factory.StartNew(() =>
-            {
-                try
-                {
-                    _completionLine = _textView.Caret.Position.BufferPosition.GetContainingLine();
-                    _completionCaretInLine = (_completionCaretPosition - _completionLine.Start);
-                    _completionText = _completionLine.GetText().Substring(0, _completionCaretInLine);
+            _completionLine = _textView.Caret.Position.BufferPosition.GetContainingLine();
+            _completionCaretInLine = (_completionCaretPosition - _completionLine.Start);
+            _completionText = _completionLine.GetText().Substring(0, _completionCaretInLine);
 
-                    StartIntelliSense(_completionLine.Start, _completionCaretPosition, _completionText);
-                }
-                catch (Exception ex)
-                {
-                    Log.Warn("Failed to start IntelliSense", ex);
-                }
-            }); 
+            try
+            {
+                StartIntelliSense(_completionLine.Start, _completionCaretPosition, _completionText);
+            }
+            catch (Exception ex)
+            {
+                Log.Warn("Failed to start IntelliSense", ex);
+            }
         }
 
         private void StartIntelliSense(int lineStartPosition, int caretPosition, string lineTextUpToCaret)
         {
-            Interlocked.Increment(ref _triggerTag);
+            long triggerTime = DateTime.UtcNow.Ticks;

```

Comment:
I didn't make this change. This should be caused by merge. Which one should we keep? @EricMSFT